internal/generate/clusterserviceversion/bases/clusterserviceversion.go

```diff
@@ -70,6 +70,9 @@ func (b ClusterServiceVersion) GetBase() (base *v1alpha1.ClusterServiceVersion,
 		base = b.makeNewBase()
 	}
 
+	// Add sdk labels to csv
+	setSDKAnnotations(base)
```

Comment:
This needs to be moved out of the `bases` package and into the CSV generator itself so the stamps aren't added to CSVs generated by `generate csv`.